ui/src/components/ConfigurationForm/ConfigurationForm.component.tsx

```diff
@@ -62,6 +69,23 @@ export const ConfigurationForm: React.FC<ConfigurationFormProps> = ({ isEmbedded
     },
   });
 
+  const handleShowPreview = useCallback(() => {
+    // 'trigger()' triggers the validation.
+    form.trigger().then(isValid => {
```

Comment:
Is it possible to rename `trigger()` to `triggerTheValidation()` and then, thanks to that, remove the comment above?

Reply:
Not really as it is the part of the `react-hook-api`